### PR TITLE
Bug: Profile Image #10

### PR DESCRIPTION
**Pull request by yujit2003**
mobile view ->
image

Desktop view ->
image

### CODE REVIEW

File: app/[username]/UserInfo.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ const UserInfo = props => {
         <>
             <div className="mx-auto flex max-w-screen-lg flex-col items-center justify-evenly gap-4 pb-6 md:flex-row md:gap-14">
                 <div className="relative size-36 flex-none rounded-full border-[4px] border-gray-600 md:size-48">
-                    <img className="rounded-full" src={props.avatarUrl} alt={props.name} />
+                    <img className="rounded-full md:h-[11.5rem] md:w-[11.4rem]" src={props.avatarUrl} alt={props.name} />
```
Comment (devXprite) [inline app/[username]/UserInfo.js:8]:
```suggestion
                    <img className="rounded-full w-full" src={props.avatarUrl} alt={props.name} />
```

I think we can easily solve this bug by just adding a `w-full` class bcz we have set size on parent div

**Review by devXprite — CHANGES_REQUESTED** [review]

**Comment by yujit2003** [comment]
@devXprite i have applied changes

**Review by devXprite — APPROVED** [review]

**Comment by devXprite** [comment]
@yujit2003  Thanks for your contributions.

**Comment by yujit2003** [comment]
Thank you this was my first contribution in GSSoC .
@devXprite 
I have knowledge about MERN Stack can you assign me another issue related to frontend/backend so that i can work on it

**Comment by yujit2003** [comment]
Could you please add label of Level 1, 2, 3 else my PR won't count in GSSoC

**Pull request merged**